docs/user/metrics/labeled_booleans.md

```diff
@@ -0,0 +1,134 @@
+# Labeled Booleans
+
+Labeled booleans are used to record different related boolean flags.
+
+## Configuration
+
+For example, you may want to record a set of flags related to accessibility (a11y).
+
+```YAML
+accessibility:
+  features:
+    type: labeled_boolean
+    description: >
+      a11y features enabled on the device. ...
+    labels:
+      - screen_reader 
+      - high_contrast
+    ...
+```
+
+> **Note:** removing or changing labels, including their order in the registry file, is permitted. Avoid reusing labels that were removed in the past. It is best practice to add documentation about removed labels to the description field so that analysts will know of their existence and meaning in historical data. Special care must be taken when changing GeckoView metrics sent through the Glean SDK, as the index of the labels is used to report Gecko data through the Glean SDK.
+
+## API
+
+Now you can use the labeled boolean from the application's code:
+
+{{#include ../../tab_header.md}}
+
+<div data-lang="Kotlin" class="tab">
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Accessibility
+Accessibility.features["screen_reader"].set(isScreenReaderEnabled())
+Accessibility.features["high_contrast"].set(isHighContrastEnabled())
+```
+
+There are test APIs available too:
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Accessibility
+// Was anything recorded?
+assertTrue(Accessibility.features["screen_reader"].testHasValue())
+assertTrue(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+assertEquals(True, Accessibility.features["screen_reader"].testGetValue())
+assertEquals(False, Accessibility.features["high_contrast"].testGetValue())
+// Did we record any invalid labels?
+assertEquals(1, Accessibility.features.testGetNumRecordedErrors(ErrorType.InvalidLabel))
+```
+
+</div>
+
+<div data-lang="Swift" class="tab">
+
+```Swift
+Accessibility.features["screen_reader"].set(isScreenReaderEnabled())
+Accessibility.features["high_contrast"].set(isHighContrastEnabled())
+```
+
+There are test APIs available too:
+
+```Swift
+@testable import Glean
+
+// Was anything recorded?
+XCTAssert(Accessibility.features["screen_reader"].testHasValue())
+XCTAssert(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+XCTAssertEqual(true, try Accessibility.features["screen_reader"].testGetValue())
+XCTAssertEqual(false, try Accessibility.features["high_contrast"].testGetValue())
```

Comment:
I wanted to complain that this doesn't have the `testGetNumRecordedErrors` line here, but I checked and we don't actually have that method on labeled metrics in Swift 🤔

We should file a bug.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1621698